frontend/src/components/WritingTable/WritingTable.tsx

```diff
@@ -0,0 +1,101 @@
+import { MediumLogoIcon, TistoryLogoIcon } from 'assets/icons';
+import { usePageNavigate } from 'hooks/usePageNavigate';
+import { Fragment } from 'react';
+import { styled } from 'styled-components';
+import { Writing } from 'types/apis/writings';
+import { Blog } from 'types/domain';
+import { dateFormatter } from 'utils/date';
+
+type Props = { writings: Writing[] };
+
+const WritingTable = ({ writings }: Props) => {
+  const { goWritingPage } = usePageNavigate();
+
+  const getPublishedToLogoIcon = (blogName: Blog) => {
+    switch (blogName) {
+      case 'MEDIUM':
+        return <MediumLogoIcon width='2.4rem' height='2.4rem' />;
+      case 'TISTORY':
+        return <TistoryLogoIcon width='2.4rem' height='2.4rem' />;
+    }
+  };
+
+  return (
+    <>
+      <S.WritingTableContainer>
+        <colgroup>
+          <col style={{ width: '60%' }} />
+          <col style={{ width: '20%' }} />
+          <col style={{ width: '20%' }} />
+        </colgroup>
+        <thead>
+          <tr>
+            <th>Title</th>
+            <th>Published To</th>
+            <th>Published Time</th>
+          </tr>
+        </thead>
+        <tbody>
+          {writings.map((writing) => (
```

Comment:
```suggestion
         {writings.map(({ id, title, publishedDetails, createdAt }) => (
```
밑에 구조분해 할당쓰는 거 어떻습니까

Reply:
오.. 좋은데요?
반영했습니다